packages/components/pie-modal/src/modal.scss

```
@@ -137,18 +150,31 @@
         --modal-button-spacing: var(--dt-spacing-d);
         --modal-footer-padding: var(--dt-spacing-d);
 
+        grid-area: footer;
+        min-block-size: var(--dt-spacing-e);
```

Would it be preferable to update the variable here instead?
```suggestion
        --modal-block-size: var(--dt-spacing-e);
```